src/prefect/infrastructure/docker.py

```diff
@@ -348,6 +369,9 @@ def _build_container_settings(
             extra_hosts=self._get_extra_hosts(docker_client),
             name=self._get_container_name(),
             volumes=self.volumes,
+            mem_limit=self.mem_limit,
+            memswap_limit=self.memswap_limit,
+            privileged=self.privileged,
```

Comment:
These 3 arguments will be added to the `host_config` at `docker.models.containers._create_container_args`.